metad.go

```diff
@@ -28,11 +28,12 @@ import (
 	"github.com/golang/gddo/httputil"
 	"github.com/gorilla/mux"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/yunify/metad-/log"
```

Comment:
what is this `github.com/yunify/metad-/log` ?

Reply:
fixed